Site_Code/script.js

```diff
@@ -0,0 +1,466 @@
+  currentTab.appendChild(cardToAppend);
+}
+function createTitlelessInfoCard() {
+  var titleCardID = guidGenerator();
+  var currentTab = document.getElementById("Grid-Tab-1");
+  var currentTemplateCard = document.getElementsByTagName("template")[8];
+  var pageBtnPrimaryButton = currentTemplateCard.content.getElementById('small-card-main-page--btn');
+  pageBtnPrimaryButton.id = titleCardID + "-mainpg--btn";
+  var cardToAppend = currentTemplateCard.content.cloneNode(true);
+  currentTab.appendChild(cardToAppend);
+  document.getElementById(titleCardID + "-mainpg--btn").addEventListener('click', function (evt) {
+    window.open("https://www.joincyberdiscovery.com/", '_blank');
+  });
+  currentTemplateCard.content.getElementById(titleCardID + "-mainpg--btn").id = 'small-card-main-page--btn';
+}
+function fixGridCardVerticalHeightAlign(classElementsName) {
+  var pageElementsArray = [];
+  var pageTextItemsBody = document.getElementsByClassName(classElementsName);
+  for (var i = 0; i < pageTextItemsBody.length; i++){
+    pageElementsArray.push(pageTextItemsBody[i].offsetHeight);
+  }
+  var maxPageDescBodyText = Math.max.apply(null, pageElementsArray) - 8;
+  maxPageDescBodyText = maxPageDescBodyText.toString();
+  for (var q = 0; q < pageTextItemsBody.length; q++) {
+    if (pageTextItemsBody[q].offsetHeight <  maxPageDescBodyText) {
+      pageTextItemsBody[q].style.height = maxPageDescBodyText + "px";
+    }
+  }
+}
+function alignContribTableSize() {
+  var pageContribList = fixContribListShownOnDom();
+  if (pageContribList != null){
+    if (pageContribList.classList.contains('generic_contrib_card_list') == true) {
+      pageContribList.style.display = "none";
+      pageContribList.style.height = getTabMaxCardHeight(16).toString() + "px";
+      pageContribList.style.display = "";
+    }
+  }
+}
+function fixAllTabsCardsVerticalHeight(){
+  switch (previousActiveTab) {
+    case 1:
+      alignContribTableSize();
+      break;
+    case 2:
+      alignContribTableSize();
+      break;
+    case 3:
+      alignContribTableSize();
+      break;
+  }
+}
+//FINISH THIS!!
+createBannerImageWideCard("Grid-Tab-1", "home_tab_welcome_banner_image");
+createMainIntroTextCard();
+createTitlelessInfoCard();
+createHomeWideLinkCard("Grid-Tab-1", "Survival Guide", "A website where we host tips, tricks and guidance for those who are attempting the Cyber Discovery programme. All these articles have been written by past participants, most of whom have progressed to the Elite stage themselves.", "cdsurvivalguide_home_card", "https://cdsurvivalguide.netlify.com/", "View");
+createHomeWideLinkCard("Grid-Tab-1", "Discord Server", "Join the largest Cyber Discovery Discord server with nearly 750 like-minded people and students who are going through the programme as well as a few SANS Institute staff members who are involved in running the programme!", "home_discord_invite_card", "http://discord.gg/Kf8n5rT", "Join");
```

Comment:
`nearly 750` Is it possible to update this automatically, or is that horrific ammount of work? We can update it manually if it's a lot of work.

Reply:
@thebeanogamer Yep, live updating will almost certainly require access to Discord's API and quite a bit of JS XHR hackery. I'd say ballpark numbers rounded to the hundred would be fine for now, but its definitely something on a wishlist.